Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -69,12 +69,7 @@
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data
-                    android:host="coronawarn.app"
-                    android:pathPrefix="/"
-                    android:scheme="https" />
-
-                <data
-                    android:host="www.coronawarn.app"
+                    android:host="e.coronawarn.app"
```

Comment:
This is not case sensitive? I'm asking because the Validator checks both

Reply:
It is not allowed to set a HOST in the manifest ,but it works with UPPER case uri